models/tridimensional/docking_validation/chimera_generate_pams.py

```diff
@@ -1,10 +1,8 @@
-from __future__ import print_function
-
-import sys
-import os
 import argparse
-import math
 import errno
+import math
+import os
+import sys
 
 import chimera
```

Comment:
I think this import is used implicitly when running it through chimera's python interpreter